templates/docs/examples/patterns/code-snippet/code-snippet-numbered.html

```diff
@@ -0,0 +1,21 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code Snippet / Numbered{% endblock %}
```

Comment:
"Code snippet" in the example names as well for consistency (same in other code snippet examples).